simtools/data_model/metadata_collector.py

```diff
@@ -115,7 +116,7 @@ def get_data_model_schema_file_name(self):
         # from data model name
         if self.data_model_name:
             self._logger.debug(f"Schema file from data model name: {self.data_model_name}")
-            return f"{simtools.constants.SCHEMA_URL}{self.data_model_name}.schema.yml"
+            return f"{files('simtools')}/schemas/model_parameters/{self.data_model_name}.schema.yml"
```

Comment:
I have a question in regard to this `files`. Does it really access the directory in simtools or it accesses the directory in the released (main) package online? I didn't look deeply at the guidelines but if the last, then I would a bit concerned, because in that case, it wouldnt be straightforward to test updating the schemas in a development branch.

Reply:
setuptools is used when running `pip install` - the package directory known by pip to access code is the one which is access by importlib.resources.files.

The system doesn't know anything about online resources, so definitely uses the local ones.